Content.Server/Atmos/EntitySystems/AirtightSystem.cs

```diff
@@ -51,9 +51,6 @@ private void OnAirtightShutdown(EntityUid uid, AirtightComponent airtight, Compo
             }
 
             SetAirblocked(airtight, false, xform);
-
-            InvalidatePosition(airtight.LastPosition.Item1, airtight.LastPosition.Item2, airtight.FixVacuum);
-            RaiseLocalEvent(new AirtightChanged(airtight));
```

Comment:
If a door is deleted does this work?

Reply:
I just tested by using the admin delete verb to remove walls and doors and things seem to depressurize properly.

`SetAirtight` above this should always result in a directed broadcast event being raised. It should also invalidate the position if it is still anchored. If it isn't, that should've been handled by the anchor event subscription

Reply:
@Zumorica I assume it's okay then